### DISCUSSION TITLE
TOC in page layout

### DISCUSSION THREAD

**Discussion by M-H-Mukit** [cotes2020/jekyll-theme-chirpy#1413] [Q&A]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I want to utilize this template for my portfolio website, encompassing all my experiences in the About page. Can I integrate a Table of Contents (TOC) into the About page? While I discovered how to use TOC for layout: post, I'm unsure about its implementation in layout: page.

### Operations you have already tried

---
# the default layout is 'page'
layout: page
toc: true
comments: false
icon: fas fa-info-circle
order: 1
---

I attempted this, but it doesn't place the Table of Contents (TOC) on the right side as intended.

**Comment by imagelessthought** [comment]
I am having a similar issue.  I made a copy of post.html layout tweaked for a new layout to use on pages I want a TOC.  The panel-heading appears, which is part of the toc.html include, but the contents are not showing up in nav id ="toc".

**Comment by steodor** [comment]
Try this: https://stackoverflow.com/a/79442651/334110 - it's a bit more involving apparently, but it can be done. You can see the result of what he did here: https://mubtasimfuad.com/resources/ . Actually, I found the initial discussion right here: https://github.com/cotes2020/jekyll-theme-chirpy/discussions/2241

**Comment by rafisics** [comment]
I have faced this issue, as @steodor already mentioned. My site template can be viewed here: https://github.com/rafisics/website-theme. 
But it should be noted I have made a couple of additional changes (e.g., updating the workflow, using `_pages` instead of `_tabs`) which somehow caused a PWA issue (https://github.com/cotes2020/jekyll-theme-chirpy/discussions/2258) that I haven't solved yet. 
So, I would suggest trying the solution of [lotusk08](https://github.com/lotusk08) first: https://github.com/cotes2020/jekyll-theme-chirpy/discussions/2241#discussioncomment-12741177
